cmd/cored/main.go

```diff
@@ -528,3 +544,8 @@ func (w *errlog) Write(p []byte) (int, error) {
 	}
 	return len(p), nil // report success for the MultiWriter
 }
+
+func defaultDir() string {
+	// TODO(kr): something in ~/Library on darwin?
+	return filepath.Join(os.Getenv("HOME"), ".cored")
```

Comment:
Now that you mention it, we set up directories through the installers for both windows and mac. We should probably just stick this in the same directory?